src/templates/UserList/UserList.tsx

```diff
@@ -0,0 +1,61 @@
+import { CustomTableToolbarProps, HeadCell, SortingRule } from '../../components/CustomTable/CustomTable.types';
+import { CustomTable } from '../../components/CustomTable/CustomTable';
+import { SortingDirection } from '../../common/utils/SortingDirection';
+import { User } from '../../common/types/componentTypes.types';
+import { UserListToolbar } from './UserListToolbar';
+import { useState } from 'react';
+
+export function UserList() {
+  const [sortingConfig, setSortingConfig] = useState<SortingRule[]>(buildCustomTableSortingConfig());
+  const [selectedRowsIds, setSelectedRowsIds] = useState<readonly string[]>([]);
+  const [page, setPage] = useState(0);
+  const [initialRows, setRows] = useState<User[]>([]);
+  const defaultRowsPerPage = 10;
+  const [rowsPerPage, setRowsPerPage] = useState(defaultRowsPerPage);
+  const url = '../src/templates/UserList/testUsers.json';
+
+  function buildCustomTableHeaderCells(): HeadCell<keyof User>[] {
+    const headerCells: HeadCell<keyof User>[] = [];
+    // eslint-disable-next-line sort-keys
+    const row = { id: '', email: '', name: '', password: '', roles: [], roomNumber: 0, surname: '' } as User;
```

Comment:
I think this casting shouldn't be there

Reply:
The keys of the rows are now bulit in a different way.